sites/avivator/src/components/Controller/components/ChannelOptions.jsx

```diff
@@ -49,12 +49,15 @@ function ChannelOptions({ handleRemoveChannel, handleColorSelect, disabled }) {
         <Paper className={classes.paper}>
           <ClickAwayListener onClickAway={toggle}>
             <MenuList id="channel-options">
-              <MenuItem dense disableGutters onClick={handleRemoveChannel}>
-                <span className={classes.span}>Remove</span>
-              </MenuItem>
               <MenuItem dense disableGutters className={classes.colors}>
                 <ColorPalette handleColorSelect={handleColorSelect} />
               </MenuItem>
+              <MenuItem dense disableGutters onClick={setFull}>
+                <span className={classes.span}>Full</span>
+              </MenuItem>
+              <MenuItem dense disableGutters onClick={setMaxMin}>
+                <span className={classes.span}>Max/Min</span>
+              </MenuItem>
```

Comment:
I don't think these need separate handlers. You can use [data-attributes](https://developer.mozilla.org/en-US/docs/Learn/HTML/Howto/Use_data_attributes#javascript_access) to access a particular value for the class (read: don't rely on visible HTML).

```javascript
function ChannelOptions({ handleColorSelect, disabled, handleModeSelect }) {
/* ... */
  function handleClick(evt) {
    const { mode } = evt.currentTarget.dataset;
    handleModeSelect(mode);
  }
/* ... */
<MenuItem data-mode="full" onClick={handleClick} dense disableGutters>
  <span className={classes.span}>Full</span>
</MenuItem>

<MenuItem data-mode="max/min" onClick={handleClick} dense disableGutters>
  <span className={classes.span}>Max/Min</span>
</MenuItem>
```

Reply:
oh rad, for sure, this looks much cleaner.